rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -147,3 +151,104 @@ def find_exactly_one_fluid_loop(rmi, loop_id):
         loop_id,
         rmi,
     )
+
+
+def has_heating_system(rmi_b, hvac_b_id):
+    """
+    Check whether heating system exists or not.
```

Comment:
You could expand on this function description:
```
Check whether the specified hvac system has a heating system.
```

Reply:
Updated!